webapp/channels/src/components/widgets/icons/entra_id_icon.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+export default function EntraIdIcon(props: React.HTMLAttributes<HTMLSpanElement>) {
+    const {formatMessage} = useIntl();
+
+    return (
+        <span {...props}>
+            <svg
+                width='16'
+                height='16'
+                viewBox='0 0 16 16'
+                fill='none'
+                xmlns='http://www.w3.org/2000/svg'
+                aria-label={formatMessage({id: 'generic_icons.login.oneLogin', defaultMessage: 'One Login Icon'})}
```

Comment:
Should this label be updated? I'm surprised we even called the old one "One Login Icon" since that seems like it was also incorrect

Reply:
Actually, this should be removed. Even if this was the correct text, it's on a button that already has text. If we changed it to "Entra ID icon", it would then cause a screen reader to read out both the icon label and the button text ("Entra ID icon Entra ID"). You can see that by looking at the Name section of the with the Accessibility tab in the Chrome dev tools

![Screenshot 2024-09-19 at 4 22 34 PM](https://github.com/user-attachments/assets/46d5609a-5925-4b73-a63e-71eeac14f07a)

I don't think we have any linters set up to require an aria-label on SVGs, but if we do, you should get around them by setting it to the empty string

Reply:
thanks @hmhealey , we don't have linters for reinforce the set up or for preventing to set it if not necessary. The aria-label was just there in the SVG, so I just kept it. Good catch!

Reply:
Unfortunately, we can't really lint for the text being mismatched, but I do want to do a presentation and make some docs about how this sort of thing should be done and how to check for it